test/examples/AriaTags-test.js

```diff
@@ -0,0 +1,16 @@
+const test = require(`tape`)
+const Yolk = require(`yolk`) // eslint-disable-line no-unused-vars
+const renderInDoc = require(`../helpers/renderInDoc`)
+
+test(`AriaTags: using aria tags`, t => {
+  t.plan(2)
+  t.timeoutAfter(100)
+
+  const component = <div aria={{hidden: false, labeledby: `parentId`}} />
+  const [node, cleanup] = renderInDoc(component)
+
+  t.equal(node.hasAttribute(`aria-hidden`), true) && t.equal(node.getAttribute(`aria-hidden`), `false`)
+  t.equal(node.hasAttribute(`aria-labeledby`), true) && t.equal(node.getAttribute(`aria-labeledby`), `parentId`)
```

Comment:
`t.equal` returns undefined so those other two conditions are never checked. The plan should be 4 not 2.